content/graph/GomoryHu.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * Author: chilli, Takanori MAEHARA
+ * Date: 2020-04-03
+ * License: CC0
+ * Source: https://github.com/spaghetti-source/algorithm/blob/master/graph/gomory_hu_tree.cc#L102
+ * Description: Given a list of edges representing an undirected flow graph, returns edges of the Gomory-Hu tree. The max flow between any pair of
+ * vertices is given by minimum edge weight along the Gomory-Hu tree path.
+ * Time: $O(V)$ Flow Computations
+ * Status: Tested on CERC 2015 J, stress-tested
+ */
+#pragma once
+
+#include "PushRelabel.h"
+
+typedef array<ll, 3> edge;
```

Comment:
I wonder if PushRelabel's Edge struct should be made internal so that this can be named `Edge`. But, eh, having the same name may still just be confusing.

Reply:
having `edge` and `Edge` is going to be confusing anyways.I just moved it in.